modules/007-registrypackages/images/kubeadm-centos/scripts/install

```diff
@@ -14,14 +14,30 @@
 # limitations under the License.
 
 set -Eeo pipefail
-  PACKAGES_TO_INSTALL=""
-if ! rpm --quiet -q "$(rpm -qip cri-tools.x86_64.rpm | grep Name | awk '{print $3}')"; then
-  PACKAGES_TO_INSTALL="cri-tools.x86_64.rpm"
+packages_to_install=""
+package_cri_tools="cri-tools"
+package_file_cri_tools="${package}.x86_64.rpm"
+rpm_name_cri_tools="$(rpm -qip ${package_file_cri_tools} 2>/dev/null | grep Name | awk '{print $3}')"
+rpm_version_cri_tools="$(rpm -qip ${package_file_cri_tools} 2>/dev/null | grep Version | awk '{print $3}')"
+
+if ! rpm --quiet -q "${rpm_name_cri_tools}-${rpm_version_cri_tools}"; then
+  packages_to_install="${packages_to_install} ${package_file_cri_tools}"
+else
+  echo "RPM ${package_file_cri_tools} already installed."
 fi
-if ! rpm --quiet -q "$(rpm -qip kubeadm.x86_64.rpm | grep Name | awk '{print $3}')"; then
-  PACKAGES_TO_INSTALL="${PACKAGES_TO_INSTALL} kubeadm.x86_64.rpm"
+
+package_kubeadm="kubeadm"
+package_file_kubeadm="${package}.x86_64.rpm"
```

Comment:
i am not looking for `package` variable, maybe here should be `package_kubeadm`?

Reply:
yes it is error, thanks